### ISSUE TITLE
Not an issue / for advice

### ISSUE THREAD

**Issue by Npa972** [Digitalone1/PulseEffects-Presets#13]
Hi thank you for your settings?

I appreciate a bit of crystallizer and extra-stereo (0.25 in output) .

In what position do you recommend placing them in your settings ?

I added these effects without moving them. They are located between the equalizer and the limiter.

Thanks for your advice.

**Comment by Digitalone1** [comment]
Yes, before the Limiter is a good position. Maybe decrease a little bit the input gain of the Crystallizer/StereoTools so that the Limiter does not "limit" them too much.

**Comment by Npa972** [comment]
Thanks you i will try.

Easyeffects + your settings really change my music especially with classical !
